application/common/models/Verification.php

```diff
@@ -0,0 +1,113 @@
+<?php
+namespace common\models;
+
+use common\models\User;
+use Sil\IdpPw\Common\PhoneVerification\NotMatchException;
+use yii\base\Model;
+
+class Verification extends Model
+{
+    const TYPE_EMAIL = 'email';
+    const TYPE_PHONE = 'phone';
+
+    /**
+     * Send email for verification
+     * @param string $toAddress
+     * @param string $subject
+     * @param string $view Full view path with alias, ex: @common/mail/reset/self
+     * @param string $code
+     * @param User $forUser
+     * @param null|string $ccAddress
+     * @param null|integer $eventLogUserId
+     * @param null|string $eventLogTopic
+     * @param null|string $eventLogDetails
+     */
+    public static function sendEmail(
+        $toAddress,
+        $subject,
+        $view,
+        $code,
+        $forUser,
+        $ccAddress = null,
+        $eventLogUserId = null,
+        $eventLogTopic = null,
+        $eventLogDetails = null
+    ) {
+        $body = \Yii::$app->mailer->render(
+            $view,
+            [
+                'idpName' => \Yii::$app->params['idpName'],
+                'name' => $forUser->first_name,
+                'code' => $code,
+                'toAddress' => $toAddress,
+            ]
+        );
+
+        EmailQueue::sendOrQueue(
+            $toAddress,
+            $subject,
+            $body,
+            $body,
```

Comment:
I'm guessing it doesn't matter, but do you know if it will cause any problems if the textBody of an email contains HTML? Some of our views contain link tags, and we're using the same body for htmlBody and textBody. Just a thought.